content/_data/changelogs/weekly.yml

```diff
@@ -1613,6 +1613,36 @@
       message: >
         Internal: Add documentation and convenience methods for the <code>User.CanonicalIdResolver</code> extension point.
     # pull: 3165 tests only (JENKINS-34855)
+- version: "2.94"
+  date: 2017-12-11
+  changes:
+    - type: rfe
+      issue: 25286
+      pull: 1437
+      message: >
+        Export <tt>assignedLabels</tt> for agents and <tt>labelExpression</tt> for applicable job types in remote API.
+    - type: rfe
+      issue: 20474
+      pull: 3177
+      message: >
+        Optimization: Don't consult the authorization strategy about whether the internal <tt>SYSTEM</tt> pseudo-user has a given permission.
+    - type: rfe
+      pull: 3111
+      message: >
+        Update SSHD Module 2.0 to 2.3 to fire authentication events in <tt>SecurityListener</tt>s when a user connects using SSH.
+      references:
+        - url: https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md
+          title: changelog
+    - type: bug
+      issue: 47439
+      pull: 3166
+      message: >
+        The setup wizard is now resumed upon restart if it hasn't been completed yet, instead of showing the regular login screen (regression in 2.81).
+    # pull: 3168 internal/build
+    # pull: 3160 too minor (one file translated to Chinese)
+    # pull: 3182 internal/build
+    # pull: 3183 internal/build (update PowerMock and Objenesis)
+    # pull: 3184 internal (GMaven cleanup)
```

Comment:
I think it worth mentioning it as a "Developer: Jenkins core does not longer support writing classes and tests in Groovy" or so.

Reply:
Given what was migrated, it looks like KK was the only one to do that anyway 😉